distributed/cli/utils.py

```diff
@@ -0,0 +1,19 @@
+py3_err_msg = """You are using Python 3 and your system is not compatible with the Unicode text model in Python 3
+This usually gets solved by exporting this environment variables but it depends on your system:
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+For more information see: http://click.pocoo.org/5/python3/
+"""
```

Comment:
This looks good to me.  Can we reverse the order of the language to say first that the terminal doesn't support unicode and second that this is an issue for Python 3?